pkg/ratelimit/limitware/middleware.go

```diff
@@ -111,10 +110,6 @@ func NewMiddleware(ctx context.Context, s limiter.Store, f httplimit.KeyFunc, op
 		return nil, fmt.Errorf("stat view registration failure: %w", err)
 	}
 
-	if err := view.Register(redis.ObservabilityMetricViews...); err != nil {
```

Comment:
Can you help me understand why we moved this out of the main package and put it throughout the server packages? I'd rather leave this in the middleware or observability packages.

Reply:
You are correct, moving inside server is unnecessary as the servers might not be using redis at all. Moved the code inside the code that uses redis.